### ISSUE TITLE
Support relative path as a reporter name

### ISSUE THREAD

**Issue by just-boris** [glenjamin/mocha-multi#65]
In the Mocha code there is the following bit of logic: https://github.com/mochajs/mocha/blob/master/lib/mocha.js#L234

it allows customers to write local reporters and provide them to mocha. This feature is even used in this package in the [verify script](https://github.com/glenjamin/mocha-multi/blob/master/verify).

However it is not working in `mocha-multi`

```
mocha -R mocha-multi --reporter-options spec=-,./my-custom.js=out.txt
```

**Comment by glenjamin** [comment]
Can you provide a full reproducible example please? It should be a matter of getting the paths right

**Comment by just-boris** [comment]
Yes, here we go: https://github.com/glenjamin/mocha-multi/pull/66

The tests there will fail with the error `ERROR: Unable to find 'fixtures/dummy-reporter.js' reporter`. The expected result that reporter can be found.
